manifest.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure">
+  <ProviderNameSpace>Microsoft.Azure.Extensions</ProviderNameSpace>
```

Comment:
You'll need to provide the following.  Send it to my internal email.

* SubscriptionID of the publishing subscription.
* Thumbprint of the management certificate for the publishing subscription.
* SPN
 * client_id
 * tenant_id
 * client_secret

You'll need to register the SPN with secret store as well as the management certificate.  Please create ones just for the CI.  Please register the client_secret and management certificate in Jenkins.  (I can help with that.)  We'll need the IDs of these Jenkins secrets.